.github/workflows/trigger-rtd.yml

```diff
@@ -0,0 +1,29 @@
+name: Trigger RTD on Versioned Updates
+
+on:
+  push:
+    branches:
+      - main
+      - 'b[0-9]+.[0-9]+'
```

Comment:
Why did you add the `b[0-9]+.[0-9]+`?

Reply:
I was following the same setup as the existing GitHub workflows for the PR push section. I figured it made sense to include it since future branches and updates would need to build their own version of the docs too. But I’m totally fine removing it if there’s a better way to handle version changes.